FBC/DebtAndEquities/CreditEvents.rdf

```diff
@@ -201,9 +207,9 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;Repudiation">
-		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;EntitySpecificCreditEvent"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;ObligationSpecificCreditEvent"/>
 		<rdfs:label xml:lang="en">repudiation</rdfs:label>
-		<skos:definition xml:lang="en">entity-specific credit event involving the dispute of the validity of a contract and refusal to honor its terms</skos:definition>
+		<skos:definition xml:lang="en">credit event involving the dispute of the validity of a contract and refusal to honor its terms</skos:definition>
```

Comment:
See https://www.investopedia.com/terms/r/repudiation.asp - repudiation may not involve a dispute "The simplest method of repudiation is when a party comes right out and admits that they are unwilling or unable to perform their obligations under the contract. A party’s conduct can also amount to an act of repudiation."